### DISCUSSION TITLE
Welcome to scarlet31.github.io Discussions!

### DISCUSSION THREAD

**Discussion by scarlet31** [scarlet31/scarlet31.github.io#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by scarlet31** [comment]
Testing.  This is a test.

**Comment by scarlet31** [comment]
Looks like it works for me. Be sure to write something here so I know you've been around.

**Comment by burtonsamograd** [comment]
I'm in.

--
The Neuromancer's Boss
"He'll never steal from me again."
2023
